packages/functions/test/oidc/getVercelOidcToken.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe, expect, it, beforeEach, afterEach } from 'vitest';
+import { getVercelOidcToken } from '../../src/oidc';
+import { randomUUID } from 'crypto';
+
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** add single root `describe`

Can you wrap all of this in a single, root `describe`?